### PR TITLE
CONSOLE-4127: replace deprecated components in QueryBrowser

### PR DESCRIPTION
**Pull request by rhamilto**
This is 2 of 2 PRs to address https://issues.redhat.com/browse/CONSOLE-4127

After

Note there wasn't a good place to test, so I threw the component on a blank page:

https://github.com/user-attachments/assets/65326866-73c8-45f2-b799-e4a3c574fe2f

### CODE REVIEW

**Comment by kyoto** [comment]
/lgtm

FYI @jgbernalp @zhuje @PeterYurkovich

Similar to https://github.com/openshift/console/pull/14138, a change like this will also be needed in the monitoring plugin (https://github.com/openshift/monitoring-plugin/blob/main/web/src/components/query-browser.tsx)

**Comment by rhamilto** [comment]
Tech debt so manually adding labels:
/label px-approved
/label docs-approved

/assign @yapei

File: frontend/packages/console-shared/src/components/query-browser/QueryBrowser.tsx [hunk 1]
```diff
@@ -158,18 +157,21 @@ const SpanControls: React.FC<SpanControlsProps> = React.memo(
...
+                  isExpanded={isOpen}
                   aria-label={t('public~graph timespan')}
-                  onToggle={setIsOpen}
                 />
-              }
-            />
+              )}
+              popperProps={{ position: 'right' }}
+            >
+              {dropdownItems}
```
Comment (rhamilto) [inline frontend/packages/console-shared/src/components/query-browser/QueryBrowser.tsx:173]: Missing `<DropdownList>`

**Review by sg00dwin — APPROVED** [review]
/lgtm

**Pull request merged**